app/src/main/res/menu/menu_main.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:title="Sign Out"
+        android:id="@+id/menu_sign_out"
```

Comment:
let's use orderInCategory, just so we remember to put that for other menu items we add.